KTLOSNaxxLoot.json

```diff
@@ -686,7 +686,7 @@
 			"wowID":"23054"
 		},
 		"Hammer of the Twisting Nether":{
-			"prio":["Doligrian(1)","Jaz(1)","Celesdea(1)","Peku(2)","Nisio(2)","Nattys(2)","Jowsus, Claris, Thrane(1)","Healers to EP/GP"],
```

Comment:
Rest looks good, but wondering why removing me on Hammer.
Is it cuz of Widow's embrace? We usually don't drop prio from things from from EPGP

Reply:
Apparently, with both Mazzo and Peku approving according to her. We should try not to let branches get too stale, there are already several merge conflicts here. If things change on Friday we can submit a new pull.